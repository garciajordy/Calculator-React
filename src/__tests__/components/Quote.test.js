import React from 'react';
import { render, cleanup, screen } from '@testing-library/react';
import Quote from '../../components/Quote';

afterEach(cleanup);

it('should take a snapshot', () => {
  const { asFragment } = render(<Quote />);
  expect(asFragment(<Quote />)).toMatchSnapshot();
});

it('Should display header information', () => {
  render(<Quote />);
  expect(screen.getByText('Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.')).toBeInTheDocument();
});
