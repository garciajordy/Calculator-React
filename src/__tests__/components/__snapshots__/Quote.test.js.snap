// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="quote"
  >
    <h2>
      Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
      <span>
        â€“William Paul Thurston
      </span>
    </h2>
  </div>
</DocumentFragment>
`;
